src/Speckle.Sdk/Serialisation/V2/Receive/DeserializeProcess.cs

```diff
@@ -16,10 +16,11 @@ public record DeserializeOptions(
 public sealed class DeserializeProcess(
   IProgress<ProgressArgs>? progress,
   IObjectLoader objectLoader,
-  IObjectDeserializerFactory objectDeserializerFactory
+  IObjectDeserializerFactory objectDeserializerFactory,
+  DeserializeProcessOptions? options = null
 ) : IDeserializeProcess
 {
-  private DeserializeOptions _options = new(false);
+  private readonly DeserializeProcessOptions _options = options ?? new(false);
 
   private readonly ConcurrentDictionary<string, (string, IReadOnlyList<string>)> _closures = new();
   private readonly ConcurrentDictionary<string, Base> _baseCache = new();
```

Comment:
We should not need to optimise for cases where we have a single c# instance of Base appear multiple times in the graph